OpenRA.Mods.Common/UpdateRules/Rules/20201213/ReplaceCrateSecondsWithTicks.cs

```diff
@@ -0,0 +1,45 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2021 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+
+namespace OpenRA.Mods.Common.UpdateRules.Rules
+{
+	public class ReplaceCrateSecondsWithTicks : UpdateRule
+	{
+		public override string Name => "Changed Crate Lifetime to Duration and use ticks and renamed PanicLength to PanicDuration.";
+
+		public override string Description =>
+			"Crate.Lifetime was the last non-sound-related place to still use 'seconds'\n" +
+			" by multiplying with 25 internally. Converted to use ticks like everything else.\n" +
```

Comment:
Were these spaces at the start of the line intentional?

They look odd in the output:
```
Found 1 API change:
  * ReplaceCrateSecondsWithTicks: Changed Crate Lifetime to Duration and use ticks and renamed PanicLength to PanicDuration.
     Crate.Lifetime was the last non-sound-related place to still use 'seconds'
      by multiplying with 25 internally. Converted to use ticks like everything else.
      Also renamed Lifetime to Duration and ScaredyCat.PanicLength to PanicDuration to match other places.
```

Reply:
Kind of, I somehow (mis?)remembered a recent PR having something similar.
Doesn't really make sense with `\n` now that I think about it, I'll change it.